spot_rl_experiments/spot_rl/envs/skill_manager.py

```diff
@@ -1,15 +1,16 @@
 # Copyright (c) Meta Platforms, Inc. and its affiliates.
 # This source code is licensed under the MIT license found in the
 # LICENSE file in the root directory of this source tree.
+# black: ignore-errors
 
-
-from typing import Any, Dict, List, Tuple
+from typing import Any, Dict, Tuple
 
 import magnum as mn
 import numpy as np
 import rospy
 from multimethod import multimethod
 from perception_and_utils.utils.generic_utils import conditional_print
+from spot_rl.envs.test_cfslam import calculate_height
```

Comment:
let's move the calculate_height stuff somewhere else, maybe within utils/utils.py? @jimmytyyang do you have any suggestions?

Reply:
agree with what you said here!